indico/modules/categories/controllers/management.py

```diff
@@ -34,3 +58,66 @@ def _category_query_options(self):
     def _process(self):
         return WPCategoryManagement.render_template('management/content.html', self.category, 'content',
                                                     categories=self.category.children)
+
+
+class RHManageCategorySettings(RHManageCategoryBase):
+    def _process(self):
+        defaults = FormDefaults(self.category,
+                                meeting_theme=self.category.default_event_themes['meeting'],
+                                lecture_theme=self.category.default_event_themes['lecture'])
+        form = CategorySettingsForm(obj=defaults, category=self.category)
+        icon_form = CategoryIconForm(obj=self.category)
+
+        if form.validate_on_submit():
+            self.category.populate_from_dict(form.data, skip={'meeting_theme', 'lecture_theme'})
+            self.category.default_event_themes = {
+                'meeting': form.data['meeting_theme'],
```

Comment:
`form.meeting_theme.data` - no need to build a dict twice just to get a value from it